client/my-sites/post-selector/selector.jsx

```diff
@@ -403,15 +403,22 @@ const PostSelectorPosts = React.createClass( {
 			'is-type-labels-visible': isTypeLabelsVisible
 		} );
 
+		const pagesToRequest = filter( requestedPages, ( page ) => {
```

Comment:
Minor: And uncertain how well it plays with `key`, but instead of iterating `requestedPages` twice (`filter`, then `map`), could the `map` return value return `null` for the first page if we're suppressing?